docs/evm-sidechain/deploy-a-bridge.md

```diff
@@ -0,0 +1,203 @@
+---
+html: deploy-a-bridge.html
+blurb: Follow these steps to deploy a bridge.
+labels:
+  - Interoperability
+status: not_enabled
+---
+# Deploy a Bridge
+
+Before deploying a bridge, familiarize yourself with the parameters and terms used:
+
+- **Currency:** The currency used by the bridge. Depending on the token, the bridge has different setups. If the currency is XRP, the door account is called a _Native Bridge Door_; if the currency is a token, the door account is called a _Token Bridge Door_.
+- **Locking Chain:** The chain where the token originally exists. If the locking chain is an XRPL chain, the bridge IOU issuer isn't the bridge door account. If the locking chain is an EVM chain, the ERC20 owner isn't the bridge door account.
```

Comment:
We try to avoid the term "IOU". Also, I'd rather tell the user what the issuer _is_ rather than what it isn't.

```suggestion
- **Locking Chain:** The chain where the token originally exists. If the locking chain is an XRPL chain, the token has an issuer that is separate from the bridge door account. If the locking chain is an EVM chain, the token has an ERC20 owner that is separate from the bridge door account.
```